node-lib/src/runner.rs

```diff
@@ -253,8 +271,40 @@ async fn start(
     .expect("Failed to prepare data directory");
     let lock_file = lock_data_dir(&data_dir)?;
 
+    if run_options.clean_data.unwrap_or(false) {
+        clean_data_dir(&data_dir, &data_dir.join(LOCK_FILE_NAME))?;
+    }
+
     log::info!("Starting with the following config:\n {node_config:#?}");
-    let (manager, controller) = initialize(chain_config, data_dir, node_config).await?;
+    let (manager, controller) = match initialize(
+        chain_config.clone(),
+        data_dir.clone(),
+        node_config.clone(),
+    )
+    .await
+    {
```

Comment:
Nitpick: Please add an empty line after await

Reply:
cannot do that because it's automatic formatting of match expression

Reply:
Never mind.